servers/cromwell/jobs/test/test_jobs_controller.py

```diff
@@ -434,7 +434,91 @@ def _request_callback(request, context):
+        """
+        workflow_id = 'id'
+        subworkflow_id = 'subworkflow_id'
+        workflow_name = 'test'
+        status = 'Failed'
+        timestamp = '2017-11-08T05:06:41.424Z'
+        response_timestamp = '2017-11-08T05:06:41.424000+00:00'
+        inputs = {'test.inputs': 'gs://project-bucket/test/inputs.txt'}
+        outputs = {
+            'test.analysis.outputs': 'gs://project-bucket/test/outputs.txt'
+        }
+        labels = {'cromwell-workflow-id': 'cromwell-12345'}
+        std_err = '/cromwell/cromwell-executions/id/call-analysis/stderr'
+        std_out = '/cromwell/cromwell-executions/id/call-analysis/stdout'
+        attempts = 1
+        return_code = 0
+
+        def _request_callback(request, context):
+            context.status_code = 200
+            return {
+                'workflowName': workflow_name,
+                'id': workflow_id,
+                'status': status,
+                'calls': { },
+                'inputs': inputs,
+                'labels': labels,
+                'outputs': outputs,
+                'submission': timestamp,
+                'end': timestamp,
+                'start': timestamp,
+                'failures': [
+                    {'causedBy': [],
+                     'message': 'Something failed'}
+                ]
+            }  # yapf: disable
+
+        cromwell_url = self.base_url + '/{id}/metadata'.format(id=workflow_id)
+        mock_request.get(cromwell_url, json=_request_callback)
+
+        response = self.client.open('/jobs/{id}'.format(id=workflow_id),
+                                    method='GET')
+        self.assertStatus(response, 200)
+        response_data = json.loads(response.data)
+        expected_data = {
+            'name': workflow_name,
+            'id': workflow_id,
+            'status': status,
+            'submission': response_timestamp,
+            'start': response_timestamp,
+            'end': response_timestamp,
+            'inputs': jobs_controller.update_key_names(inputs),
+            'outputs': jobs_controller.update_key_names(outputs),
+            'labels': labels,
+            'extensions':{
+                'tasks': [
+                    # {
+                    #     'name': 'analysis',
+                    #     'executionStatus': 'Succeeded',
+                    #     'start': response_timestamp,
+                    #     'end': response_timestamp,
+                    #     'stderr': std_err,
+                    #     'stdout': std_out,
+                    #     'callCached': False,
+                    #     'inputs': jobs_controller.update_key_names(inputs),
+                    #     'outputs': jobs_controller.update_key_names(outputs),
+                    #     'returnCode': return_code,
+                    #     'attempts': attempts,
+                    #     'jobId': subworkflow_id
+                    # }
+                ]
```

Comment:
Why is this commented out?

Reply:
Artifact of me trying to copy/paste a previous test. Will remove